Assets/MixedRealityToolkit-SDK/Profiles/MixedRealityControllerConfigurationProfile.md

```diff
@@ -0,0 +1,132 @@
+# Mixed Reality Controller Configuration Profile
+
+When you need to use input controllers for your Mixed Reality project, they are registered and configured centrally within the Controller configuration profile as you can see here:
+
+![](/External/HowTo/ControllerConfigurationProfile/01-MixedRealityControllerConfigurationProfileInspector.png)
+
+This enables you to very quickly define which SDK's / controllers you want to support in your project and configure how each are intended to work.
+
+The configuration is broken down in to several key components, as detailed below:
+
+## Main Controller Template definition
+---
+![](/External/HowTo/ControllerConfigurationProfile/02-ControllerTemplateDefinition.png)
+
+In the first section of the configuration, the options are detailed as follows:
+
+### Render Motion Controllers
+This defines whether or not controllers should be rendered or not.  By not adding a ControllerVisualizer in to the scene also has the same effect.
+> This also disabled the Global Model options below.
+
+### Use Default Models (for future implementation)
+This option will use the models for the controller direct from the SDK (where available), so that you don't have to configure a custom model.
+
+### Global Left / Right hand models
+These allow you at the top level to define the default model that will be drawn for each controlling hand.  If no hand is available, the Left most model will be used by default.
+
+Additionally, you can provide position and rotational offsets to affect how the model is drawn, to more accurately represent how you want the model drawn based on the controllers position given by the SDK.
+
+> note, Controller rotation offset values are entered as Quaternions for the Alpha.  We may review this going forward.  Use the [this site](https://quaternions.online/) as an aid to turn Euler rotations to Quaternions and ALWAYS remember W = 1 :D
```

Comment:
Why quarternions? Settings in the inspector should definitely be something that actually makes sense to the dev without directing them to some other site. I don't see a benefit to forcing a dev to convert their Euler rotation into a quaternion.

Reply:
We don't have to do that. I already changed the Pose Property drawer to use Euler.

Reply:
Darn you @StephenHodgson , you told me it wouldn't work/ Right changing it back!!!